docker/Dockerfile.aarch64

```diff
@@ -58,7 +58,7 @@ RUN cargo build --release \
     --features scylladb \
     && strip target/aarch64-unknown-linux-gnu/release/linera \
     && strip target/aarch64-unknown-linux-gnu/release/linera-proxy \
-    && strip target/aarch64-unknown-linux-gnu/release/linera-server
+    && strip target/aarch64-unknown-linux-gnu/release/linera-server \
```

Comment:
@christos-h @andresilva91 IIRC one wouldn't need `\` if `&&` was at the end of the line.

Reply:
Yeah, you're right. There was this bash formatter that I was using that preferred it this way for some reason, so I just left it like this 😅
We could change it though, if you prefer the other way

Reply:
That was just FYI. Keep using your favorite formatter! :)